packages/noya-designsystem/src/components/GridView.tsx

```diff
@@ -0,0 +1,103 @@
+import { memo, useCallback, ReactNode } from 'react';
+import styled from 'styled-components';
+
+const Grid = styled.div(({ theme }) => ({
+  flex: 1,
+  color: theme.colors.text,
+  display: 'grid',
+  padding: '14px',
+  gridTemplateColumns: 'repeat(auto-fill, 200px)',
+  gridTemplateRows: 'repeat(auto-fill, 180px)', // Makes it go
+  justifyContent: 'space-between',
+  gap: '20px',
+  overflowY: 'scroll',
```

Comment:
We can use `overflowY: 'auto'` here - this will conditionally show a scrollbar only if needed. `'scroll'` shows a scrollbar all the time, which is usually not what we want